tests/integration/test_full_pipeline.py

```
@@ -0,0 +1,410 @@
+"""
+Integration tests for the full NEEX Legal Contract Review pipeline.
+"""
+import pytest
+from pathlib import Path
+import json
+import time
+from unittest.mock import patch, MagicMock
+
+from src.core.review_orchestrator import ReviewOrchestrator
+from src.core.analysis_context import AnalysisContext
+from src.core.contract_parser import ContractParser, Clause
```

Ruff (0.12.2)

`src.core.contract_parser.Clause` imported but unused

Remove unused import: `src.core.contract_parser.Clause`

(F401)